rundeckapp/grails-app/services/rundeck/services/ExecutionService.groovy

```diff
@@ -1248,6 +1256,51 @@ class ExecutionService implements ApplicationContextAware, StepExecutor, NodeSte
         }
     }
 
+    private String getOutputLimit(ScheduledExecution scheduledExecution) {
+        String outputLimit = ""
+        String globalLimit = configurationService.getString(LoggingThreshold.EXECUTION_LOGS_OUTPUT_LIMIT)
+        String jobLimit = scheduledExecution?.logOutputThreshold
+
+        if (!globalLimit) {
+            outputLimit = jobLimit
+        }
+        else if (globalLimit && jobLimit) {
+            outputLimit = String.valueOf(Math.min(Long.valueOf(globalLimit), Long.valueOf(jobLimit)))
```

Comment:
this won't work if the limit is e.g. `1MB/node`.  the comparison is better done inside LoggingThreshold.   Load the job LoggingThreshold, and the global LoggingThreshold (if present), and compare the two.

Reply:
Done